lib/common/operations.c

```diff
@@ -498,7 +498,8 @@ pcmk_is_probe(const char *task, guint interval)
         return false;
     }
 
-    return (interval == 0) && pcmk__str_eq(task, CRMD_ACTION_STATUS, pcmk__str_none);
+    return (interval == 0)
+           && pcmk__str_eq(task, PCMK_ACTION_MONITOR, pcmk__str_none);
```

Comment:
We haven't really established a convention, but we often use literal strings in unit tests, sometimes to avoid includes, sometimes to make the test independent of any preprocessor code (though it's an open question whether that's better or worse)